gui/create-item.lua

```diff
@@ -217,15 +431,23 @@ function hackWish(unit)
      createItem({mattype,matindex},{itemtype,itemsubtype},quality,unit,description,amount)
     else
      for i=1,amount do
-      createItem({mattype,matindex},{itemtype,itemsubtype},quality,unit,description,1)
+	  if df.item_type[itemtype]~='CORPSEPIECE' then
```

Comment:
Not required, but you can also use either of these options, which are probably a bit faster, and the second one is less prone to typos:
```suggestion
	  if itemtype == df.item_type.CORPSEPIECE then
```

```suggestion
	  if df.item_corpsepiecest:is_instance(item)
```
Actually, the second one might not work if you don't have the item yet...

Reply:
edited this specific check to check the item type _before_ the loop instead of checking the item type every iteration cause it never changes (right?)